apps/tailwind-components/pages/Form.story.vue

```diff
@@ -142,22 +143,64 @@ watch(
   },
   { immediate: true }
 );
+
+const numberOfFieldsWithErrors = computed(
+  () => Object.values(errorMap.value).filter((error) => error.length > 0).length
+);
+
+const numberOfRequiredFields = computed(() =>
+  metadata.value !== null
+    ? metadata.value?.columns.filter((column) => column.required).length
+    : 0
+);
+
+const numberOfRequiredFieldsWithData = computed(() =>
+  metadata.value !== null
+    ? metadata.value?.columns.filter(
+        (column) => column.required && formValues.value[column.id]
+      ).length
+    : 0
+);
+
+const activeChapterId = ref<string>("_scroll_to_top");
+const errorMap = ref<Record<columnId, string>>({});
+
+function setActive(id: string) {
+  activeChapterId.value = id;
+}
+
+const sections = useSections(metadata, activeChapterId, errorMap);
 </script>
 
 <template>
   <div class="flex flex-row">
-    <div class="2/3 p-8 border-l">
-      <FormFields
-        id="forms-story"
-        v-if="schemaId && tableMeta && status === 'success'"
-        ref="formFields"
-        :schemaId="schemaId"
-        :metadata="tableMeta"
-        v-model="formValues"
-        @error="onErrors($event)"
+    <div class="p-8 grow flex flex-row">
+      <Legend
```

Comment:
why would we want to duplicate all this in a story? we can have a component that has all this right?